sdk/packages/react/src/hooks/useQuote.ts

```diff
@@ -62,6 +70,7 @@ export function useQuote(params: UseQuoteParams): UseQuoteResult {
     queryKey: ['quote', request],
     queryFn: async () => doQuote(apiBaseUrl, request),
     enabled,
+    ...params.queryOpts,
   })
```

Comment:
Considering `queryOpts` is optional, I think we need to ensure it's an object before using the spread, ex `...(params.queryOpts ?? {})`.
Also, I'd put these options spread before the other parameters in the object to ensure they don't accidentally override the other parameters, ex `useQuery({ ...opts, queryFn: ... })`